lib/fourOhFour.js

```diff
@@ -58,6 +58,24 @@ function fourOhFour (options) {
     })
   }
 
+  function onBadUrl (path, req, res) {
+    const { url, method } = req
+    const message = `Route ${method}:${url} not found`
+    const body = `{"error":"Not Found","message":"${message}","statusCode":404}`
+
+    // simulate normal route logging
+    const id = genReqId(req)
```

Comment:
I would like to have some feedback about the logging. Do we have a better approach for this one?